app/controllers/components/course/assessments_component.rb

```diff
@@ -7,6 +7,10 @@ def self.display_name
     I18n.t('components.assessments.name')
   end
 
+  def self.lesson_plan_item_actable_names
+    ['Course::Assessment']
```

Comment:
Suggest to use `Course::Assessment.name` instead, this will ensure that the class name is not spelt wrongly.

Reply:
Fixed for all the others also.